spec/clients_spec.rb

```diff
@@ -130,4 +130,149 @@
       expect(resp[:usage]).to eq(10)
     end
   end
+
+  context 'when calling request helpers' do
+    let(:config) { YAML.load_file('spec/config.example.yaml') }
+    subject(:client) { AppFigures::Client.new(username:   config['username'],
+                                              password:   config['password'],
+                                              app_key:    config['app_key'],
+                                              app_secret: config['app_secret']) }
+    before(:each) do
+      @af = stub_request(:any, /#{AppFigures::API::BASE_URL}\/*/)
+      @date = '2016-01-01'
+    end
+
+    it 'get usage' do
```

Comment:
For test cases, using `it '...' do` should describe the target outcome.
For example, this test expect a valid response from the API call so we can write
```
it 'returns valid body with #usage' do
...
end
```